.gitlab-ci.yml

```diff
@@ -9,11 +9,10 @@ include:
     file: github-project-pipelines.gitlab-ci.yml
 
 variables:
-  GIT_STRATEGY: none
+  CMAKE_BUILD_PARALLEL_LEVEL: 3
+  CTEST_PARALLEL_LEVEL: 3
 
 stages:
-  - clean workspace
-  - prerequisites
   - build
 
 upload_logs_to_github:
```

Comment:
We probably don't need to explicitly have this entry (setting `GIT_STRATEGY`) any more, right?

Reply:
Good catch, will remove.